setup.cfg

```diff
@@ -27,7 +27,6 @@ addopts =
   --instafail
   --profile
   --reorder 'requirements.txt' 'test-requirements.txt' '*'
-  --ignore=tests/collecting/collectors_test_dir/bears/incorrect_bear.py
```

Comment:
this is a bug I have solved in a moban template and am waiting on the mobans repo to be unbroken

Reply:
This bug fix has been solved